### ISSUE TITLE
mail-iso-2022-jp could  not support mail 2.8.1

### ISSUE THREAD

**Issue by quy-dn** [kuroda/mail-iso-2022-jp#27]
I use mail-iso-2022-jp  with redmine 5.1.3 . 
 So I got following bundler error because iso-2022-jp supports until mail 2.7.1. Could you support mail 2.8.1?

```
Could not find compatible versions
Because mail-iso-2022-jp >=2.0.9 depends on mail >= 2.2.6, <=2.7.1
  and Gemfile depends on mail ~>2.8.1
  mail-iso-2022-jp >=2.0.9 cannot be used.
So, because Gemfile depends on  mail-iso-2022-jp =2.0.9,
  version solving has failed.
```

**Comment by kuroda** [comment]
@quy-dn 

To be honest, I do not intend to devote time to maintain `mail-iso-2022-jp` as I myself am currently away from Ruby development. Sorry.

But I can review and merge your PR.

**Comment by Hieu1998** [comment]
@kuroda 
I created pull requests for fix mail 2.8.1. Please review.
https://github.com/kuroda/mail-iso-2022-jp/pull/28

**Comment by kuroda** [comment]
@Hieu1998 

Thanks. Your PR has been merged and a new version has been released.
